Clinical-Genomics/scout
# Splice junctions view

This PR adds a functionality:
- Adds 2 types of tracks to update case individuals (fix #2439) with:
  -  splice_junctions_bed
  -  rna_coverage_bigwig
- Adds a button in variants page with a link to a splice junctions view (if at least one individual has this type of tracks) (fix #2344):
![image](https://user-images.githubusercontent.com/28093618/112665239-217de900-8e5b-11eb-97d1-bbbd4a7365fe.png)

- Displays the sashimi plot with these tracks in a new window 

![image](https://user-images.githubusercontent.com/28093618/113984427-62a6cd80-984b-11eb-9f4c-f1a09eb9205b.png)


**How to test, locally**:
1. Download the demo track data from here: https://figshare.com/articles/dataset/Splice_Junctions_test_data/14381657
1. Update one or more individuals of the test case with the bigwig track:
```scout --demo update individual -c internal_id -n NA12882 rna_coverage_bigwig /path_to/Downloads/Downloads/junctions_test_data/sampleA.chr15-92835700-93031800.bigWig```
1. Update the same individual of the test case with the bed track:
```scout --demo update individual -c internal_id -n NA12882 splice_junctions_bed /path_to/Downloads/junctions_test_data/sampleA.chr15-92835700-93031800.SJ.out.bed.gz ```
1. Go to any variant of the case and click on the splice junctions link: you won't see any data because the demo tracks contain data only in the interval: **chr15:92835700-93031800**. Navigate to these coordinates and you'll see the sashimi plot

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
